packages/models/src/appMapBuilder/eventStack.js

```diff
@@ -45,10 +45,6 @@ export default class EventStack {
           parent.children.push(call);
           call.parent = parent;
         }
-      } else {
-        throw new Error(
-          `return #${event.id} is missing call #${event.parent_id}`
-        );
```

Comment:
Is this occurring?

Reply:
@dustinbyrne  We need to remove this exception because when we filter our events that exception blocks the whole script runtime.
We filter out only calls from events:
https://github.com/applandinc/appmap-js/blob/25ab8e0e451f93e469850a06eafb6a256620fce6/packages/components/src/pages/VsCodeExtension.vue#L538

Reply:
That would indicate an issue in the filtering though, wouldn't it? This exception is thrown when a return event has no matching call. If the call is being filtered out, the return should be filtered out as well.